src/components/personalRequests/PreviousRequests.tsx

```diff
@@ -0,0 +1,71 @@
+import Title from 'antd/lib/typography/Title';
+import React from 'react';
+import styled from 'styled-components';
+import dateFormat from 'dateformat';
+import { PersonalRequest } from '../../containers/personalRequests/ducks/types';
+import { ORANGE } from '../../utils/colors';
+
+export interface PreviousRequestsProps {
+    requests: PersonalRequest[]
+}
+
+const Wrapper = styled.div`
+    display: flex;
+    justify-content: center;
+    margin-top: 20px;
+`;
+
+const StyledTitle = styled(Title)`
+    font-size: 25px !important;
```

Comment:
👀 in general we want to avoid !important. I know you did this because AntD wasnt working though. I would say im "ok" with this, but we should avoid this in the future if possible.